packages/cli/src/scaling/__tests__/job-processor.service.test.ts

```diff
@@ -80,4 +82,57 @@ describe('JobProcessor', () => {
 			expect(manualExecutionService.runManually).toHaveBeenCalledTimes(1);
 		},
 	);
+
+	it('should pass additional data for partial executions to run', async () => {
```

Comment:
I left the tests that were added, even though the main functionality has not changed, since there was no coverage before.